### DISCUSSION TITLE
Add a global setting for  alerts threshholds

### DISCUSSION THREAD

**Discussion by yurividal** [henrygd/beszel#185] [Ideas]
Instead of making the user set the thresholds for alerts for every server, would it be possible to have a "global" setting, that would apply to all servers, and maybe allow it to b overwritten on each server if needed?

**Comment by henrygd** [comment]
Please see issue #123 for an example of how to do this with the REST API.

As of now it's not built into the web UI, but I may add it in the future.

**Comment by henrygd** [comment]
Update: you can set alerts for all systems at once now, and optionally overwrite existing alerts.
